mltsp/transformation.py

```diff
@@ -0,0 +1,45 @@
+import os
+import numpy as np
+from .cfg import config
+from mltsp import data_management
+from mltsp import time_series as tslib
+from sklearn import model_selection
+
+
+def transform_ts_files(input_paths, dataset_keys, transform_type):
```

Comment:
We should move away from using file handles, and operate on TimeSeries where possible.